assets/quick-order-list.css

```diff
@@ -142,10 +127,6 @@ quick-order-list .quantity__button {
     display: flex;
   }
 
-  .quick-order-list__table .variant-item__inner.variant-item__inner--no-sku {
-    vertical-align: middle;
-  }
-
```

Comment:
The spacing on mobile needs to be adjusted

https://screenshot.click/29-07-d5hps-zcx8m.png